README.md

```diff
@@ -1,7 +1,9 @@
 # ebau-gwr
 
-[![Build Status](https://travis-ci.com/adfinis/ebau-gwr.svg?branch=master)](https://travis-ci.com/adfinis/ebau-gwr)
-[![Pyup](https://pyup.io/repos/github/adfinis/ebau-gwr/shield.svg)](https://pyup.io/account/repos/github/adfinis/ebau-gwr/)
+**!!!WIP!!!**
+
+[![Build Status](https://github.com/adfinis/ebau-gwr/workflows/Tests/badge.svg)](https://github.com/adfinis/ebau-gwr/actions?query=workflow%3ATests)
+[![Coverage](https://img.shields.io/badge/coverage-100%25-brightgreen.svg)](https://github.com/adfinis/ebau-gwr/blob/master/setup.cfg#L54)
```

Comment:
```suggestion
[![Coverage](https://img.shields.io/badge/coverage-100%25-brightgreen.svg)](https://github.com/adfinis/ebau-gwr/blob/main/setup.cfg#L54)
```

Not sure however if a "naive" green badge is very useful

Reply:
Fixed the wrong branch names.

> Not sure however if a "naive" green badge is very useful

It's the same in all our repos. Do you have a suggestion?

Reply:
Not really, as we enforce 100% coverage anyway :-)